Ginger/Ginger/ValueExpression/ValueExpressionEditorPage.xaml.cs

```diff
@@ -161,16 +160,19 @@ private void BuildItemsTree()
                 }                
             }
 
-
+            TreeViewItem dataRoot = AddOrGetCategory("Data");
+            dataRoot.IsExpanded = true;
         }
 
     
 
         private void AddGlobalParameters()
         {
+            TreeViewItem Parent = AddOrGetCategory("Data");
+            //TreeViewItem child = AddOrGetSubCategory("Models Global Parameters", Parent);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6356765)